CHANGELOG.md

```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2019.02.0
+
+### Added
+- Classes needed by DQN: epsilon greedy strategy, discrete MLP model and
+  discrete qf-derived policy
+- Base Model class
+- Dump plots generated with Matplotlib in TensorBoard
+- Relative Entropy Policy Search (REPS) class
+- GaussianConvBaseline and GaussianConvRegressor class
+- Dockerfiles for garage
+- Vanilla loss for NPO
+- Truncated Natural Policy Gradient (TNPG) class
+- Episodic Reward Weighted Regression (eRWR) class
+- gym.Env wrappers used for pixel environments
+- Convolutional Neural Network class
+
+### Changed
+- Move dependencies from environment.yml to setup.py
+- Update dependencies:
+  - tensorflow-probability to 0.5.x
+  - dm_control to commit 92f9913
+  - TensorFlow to 1.12
+  - MuJoCo to 2.0
+  - gym to 0.10.11
+- Move dm_control tests into the unit test tree
+- Use GitHub standard .gitignore
+- Improve the implementation of RandomizedEnv (Dynamics Randomization)
+- Decouple tensorboard from logger
+- Move files from garage/misc/instrument to garage/experiment
+
+### Removed
+- Move some garage subpackages into their own repositories:
+  - garage.viskit to rlworkgroup/viskit
+  - garage.spaces to rlworkgroup/akro
+- Theano branch
+- Some environments which duplicated openai/gym
+- Some dead files from garage/misc (meta.py and viewer2d.py)
+- Remove all code coverage tracking providers except CodeCov
+
+### Fixed
+- Clean up warnings in the test suite
+- Pickling bug in GaussianMLPolicyWithModel
+- Namescope in LbfgsOptimizer
+- Correctly sample paths in OffPolicyVectorizedSampler
+- Implementation bugs in tf/VPG
+- Bug when importing Box
+- Bug in test_benchmark_her
+
+
```

Comment:
should we also add the 2018.10.1 changelog?

Reply:
All the back ported changes have been include here, and even one of them doesn't apply anymore (fix mutually exclusive imports of Theano and TensorFlow), so I think it's better to not add it to keep things simple.

Reply:
hmm i think we still need to include v2018.10.1 here though. people should be aware that such a release exits. you should be able to just copy/paste